README.md

```diff
@@ -94,73 +94,80 @@ The plugin comes with the following defaults:
     },
   },
 
-  -- file-type specific options
-  -- any options that are passed here will override the default config
+  -- file type specific opts
+  -- any opts that are passed here will override the default config
   -- for instance, setting use_absolute_path = true for markdown will
   -- only enable that for this particular file type
   -- the key (e.g. "markdown") is the filetype (output of "set filetype?")
-
-  markdown = {
-    url_encode_path = true,
-    template = "![$CURSOR]($FILE_PATH)",
-
-    drag_and_drop = {
-      download_images = false,
+  filetypes = {
+    markdown = {
+      url_encode_path = true,
+      template = "![$CURSOR]($FILE_PATH)",
+
+      drag_and_drop = {
+        download_images = false,
+      },
     },
-  },
 
-  html = {
-    template = '<img src="$FILE_PATH" alt="$CURSOR">',
-  },
+    html = {
+      template = '<img src="$FILE_PATH" alt="$CURSOR">',
+    },
 
-  tex = {
-    relative_template_path = false,
-    template = [[
+    tex = {
+      relative_template_path = false,
+      template = [[
 \begin{figure}[h]
   \centering
   \includegraphics[width=0.8\textwidth]{$FILE_PATH}
   \caption{$CURSOR}
   \label{fig:$LABEL}
 \end{figure}
     ]],
-  },
+    },
 
-  typst = {
-    template = [[
+    typst = {
+      template = [[
 #figure(
   image("$FILE_PATH", width: 80%),
   caption: [$CURSOR],
 ) <fig-$LABEL>
     ]],
-  },
+    },
 
-  rst = {
-    template = [[
+    rst = {
+      template = [[
 .. image:: $FILE_PATH
    :alt: $CURSOR
    :width: 80%
     ]],
-  },
+    },
 
-  asciidoc = {
-    template = 'image::$FILE_PATH[width=80%, alt="$CURSOR"]',
-  },
+    asciidoc = {
+      template = 'image::$FILE_PATH[width=80%, alt="$CURSOR"]',
+    },
 
-  org = {
-    template = [=[
+    org = {
+      template = [=[
 #+BEGIN_FIGURE
 [[file:$FILE_PATH]]
 #+CAPTION: $CURSOR
 #+NAME: fig:$LABEL
 #+END_FIGURE
     ]=],
+    },
   },
+
+  -- override opts for specific files, dirs or custom triggers
```

Comment:
options?

I guess you are consistent, and I definitely prefer consistency in general. But I think it's more clear when we avoid shortening things unnecessarily.